media/css/firefox/unfck.scss

```diff
@@ -197,6 +197,23 @@ main {
         }
     }
 
+    a.mzp-c-button.mzp-t-dark.mzp-t-secondary {
+        border-color: $color-green-30;
+        color: $color-green-30;
+
+        &:hover,
+        &:focus {
+            background-color: $color-green-50;
+            border-color: $color-green-50;
+            color: $color-ink-90;
+        }
+
+        &:focus {
+            box-shadow: $field-focus-ring-dark;
+            color: $color-ink-90;
+        }
+    }
+
     .fx-privacy-link {
```

Comment:
We should really show the privacy link here imho, as it's likely a legal requirement?

Reply:
It's not in the global nav and I remember Jennifer talking to legal and getting the okay to leave it off in other places. Do you think we need to check with someone?